File: Level_2/124 나라의 숫자/Number_124.js
function solution(n) {
    let answer = "";
    let number = ["4", "1", "2"];
    
    while(n  > 0){
        answer = number[n % 3] + answer;
        n = Math.floor((n-1) / 3);
    }
    
    
    return answer;
}

File: Level_2/124 나라의 숫자/Number_124.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int n) {
    string answer = "";
    string num[] = {"4", "1", "2"};
    
    while(n){
        answer = num[n % 3] + answer;
        n = (n - 1) / 3;
    }
    
    
    return answer;
}
